plugin/evm/limitorders/memory_database.go

```diff
@@ -658,7 +681,7 @@ func getAvailableMargin(trader *Trader, pendingFunding *big.Int, oraclePrices ma
 }
 
 // deepCopyOrder deep copies the LimitOrder struct
-func deepCopyOrder(order LimitOrder) LimitOrder {
+func deepCopyOrder(order *LimitOrder) LimitOrder {
```

Comment:
what does this change?

Reply:
for this function the callee has the pointer to order struct, and earlier it had to convert it to struct before passing to this function. This way it reads better